lib/data-helpers/custom-parameter.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const { struct } = require('@janiscommerce/superstruct');
+const ApiListError = require('../api-list-error');
+
+const { mapperFilter } = require('../helpers/mapper-filter');
+
+const apiCustomParameterStruct = {
```

Comment:
⚠️ Ojo, este struct tiene que validar mejor porque en el caso de los customParameters, estos llegan en el `this.data` junto con los de filtrado y ordenamiento de la API, por lo que tenemos que hacer que el struct valide que de todos los customParameters que puedan llegar **no sean nombres reservados para otros parametros de la API**

Por ejemplo, si alguien quiere meter un custom parameter que es `sortBy` estaria pisando el `sortBy` de la API y seguramente rompan las validaciones de los otros data-helpers.

Los parametros reservados de la API list que puedan llegar en el `this.data` son `filters`, `sortBy` y `sortDirection` 👀